### PR TITLE
JSON last comma fix and removed unused variables

### PR DESCRIPTION
**Pull request by ipapapa**

### CODE REVIEW

File: src/dyn_stats.c [hunk 1]
```diff
@@ -451,7 +449,7 @@ stats_add_string(struct stats_buffer *buf, struct string *key, struct string *va
 }
 
 static rstatus_t
-stats_add_num(struct stats_buffer *buf, struct string *key, int64_t val)
+stats_add_num_last(struct stats_buffer *buf, struct string *key, int64_t val, short last)
```
Comment (shailesh33) [inline src/dyn_stats.c]: the param should be bool/boolean

File: src/dyn_stats.c [hunk 2]
```diff
@@ -472,6 +476,18 @@ stats_add_num(struct stats_buffer *buf, struct string *key, int64_t val)
     return DN_OK;
 }
 
+
+static rstatus_t
+stats_add_num(struct stats_buffer *buf, struct string *key, int64_t val)
+{
+	if (stats_add_num_last(buf,key, val, 0) == DN_ERROR) {
```
Comment (shailesh33) [inline src/dyn_stats.c]: use false, not 0

Comment (ipapapa) [inline src/dyn_stats.c]: done

File: src/dyn_stats.c (hunk 1 above)
Comment (ipapapa) [inline src/dyn_stats.c]: done

**Pull request merged**